user_data/emqx_init.sh

```diff
@@ -126,6 +126,9 @@ rate_limit {
   conn_bytes_in = infinity
 }
 
+listeners.tcp.default {
+  max_connections = 1000000000
```

Comment:
I believe there's a line for this config [above](https://github.com/emqx/cdk-emqx-cluster/blob/f853cde2d02079aa2212fcf8363d40a9209bcd5a/user_data/emqx_init.sh#L74).